dotCMS/src/main/java/com/dotcms/metrics/MetricsAPI.java

```diff
@@ -0,0 +1,67 @@
+package com.dotcms.metrics;
+
+import com.dotcms.analytics.app.AnalyticsApp;
+import com.dotcms.http.request.StringPayloadHttpRequest;
```

Comment:
I would keep anything related to http requests outside of the scope of the API.

Reply:
Refactored.